### DISCUSSION TITLE
How to enable winbar for python

### DISCUSSION THREAD

**Discussion by LumenYoung** [LunarVim/LunarVim#3472] [Q&A]
Hi, everyone,

as stated in the title. I have problem when I find that I can have winbar for the lua file while it is not working in python. I wonder if this is by design or how can I enable that for python. 

This is in python, I can only have filename
![image](https://user-images.githubusercontent.com/45258158/201625787-3dc50b10-775b-400a-a2bc-a6200e938d7b.png)

This is in lua
![image](https://user-images.githubusercontent.com/45258158/201625909-e8f109e3-02c5-49aa-8130-160babf5698b.png)

I wonder if that can be some problem with my configuration? Does it work for python in your configuration? Or which plugin from LunarVim core controls this winbar. Any of the above information would be helpful.

Thanks,
Jiaye.

**Comment by uelei** [comment]
mine works with python 
![Screenshot from 2022-11-14 19-09-23](https://user-images.githubusercontent.com/5301829/201778814-1e53274f-ddd9-42a8-adac-e9024feb743a.png)

try update your lunarvim, you can check my config as well https://github.com/uelei/dotfiles/tree/master/lvim/.config/lvim

**Reply by LumenYoung** [reply]
Thanks. But I found that It only happens for python files. So I think maybe there are something wrong with my python ftplugin. Do you know which plugin controls this winbar? I think that will be helpful for me to debug.

**Reply by uelei** [reply]
It is natively on neovim 0.8 https://github.com/neovim/neovim/releases/tag/v0.8.0

**Comment by LostInTheLogs** [comment]
do you have the lsp running? the plugin is `navic`

**Reply by LumenYoung** [reply]
> do you have the lsp running? the plugin is `navic`

Thanks, I found the plugin that offers the winbar. I disabled it and use lspsaga instead.
